packages/rich-text/src/to-html-string.js

```diff
@@ -18,13 +18,18 @@ import { toTree } from './to-tree';
  * Create an HTML string from a Rich Text value. If a `multilineTag` is
  * provided, text separated by two new lines will be wrapped in it.
  *
- * @param {Object} value        Rich text value.
- * @param {string} multilineTag Multiline tag.
+ * @param {Object} value                Rich text value.
+ * @param {string} multilineTag         Multiline tag.
+ * @param {Array}  multilineWrapperTags Tags where lines can be found if nesting
+ *                                      is possible.
  *
  * @return {string} HTML string.
  */
-export function toHTMLString( value, multilineTag ) {
-	const tree = toTree( value, multilineTag, {
+export function toHTMLString( value, multilineTag, multilineWrapperTags ) {
```

Comment:
`toDOM( { value, multilineTag, multilineWrapperTags } );` ✅ 
`toTree( { value, multilineTag, multilineWrapperTags } );` ✅ 
`toHTMLString( value, multilineTag, multilineWrapperTags );` 🤔

Reply:
The problem is that `toHTMLString` is now a public API. Fine to change it though, and make sure everything keeps working.